src/type/type.js

```diff
@@ -162,10 +162,15 @@ function getTypes(graffitiModels) {
   return types;
 }
 
+function addType(name, type) {
```

Comment:
Wasn't aware of that. Functions get hoisted, so shouldn't be an issue. Silly eslint!